README.md

```diff
@@ -120,7 +120,7 @@ Parameter | Description | Default
 -|-|-
 `kafka.topic` | Topic to write to | 
 `max.interval` | Max interval between messages (ms) | 500
-`iterations` | Number of messages to send, or less than 1 for unlimited | -1
+`iterations` | Number of messages to send from each task, or less than 1 for unlimited | -1
```

Comment:
Can we also update the `DatagenConnectorConfig.ITERATIONS_DOC` field with this?

Reply:
this is the more important place to add them, oops